test/mitmproxy/net/test_tls.py

```diff
@@ -52,6 +49,7 @@ def test_create_logfun(self):
             tls.MasterSecretLogger.create_logfun("test"),
             tls.MasterSecretLogger)
         assert not tls.MasterSecretLogger.create_logfun(False)
+"""
```

Comment:
just temporary - or whats the plan?

Reply:
Yes. I have thoroughly cleaned up the TLS implementation (draft PR in a second), but that currently depends on a cryptography release followed by a pyOpenSSL PR (https://github.com/pyca/pyopenssl/pull/985) followed by a pyOpenSSL release.